test/interface-tests.spec.js

```diff
@@ -542,26 +268,57 @@ describe('kubo-rpc-client tests against go-ipfs', () => {
       args: ['--enable-pubsub-experiment']
     }
   }), {
-    skip: [{
-      name: 'should receive messages from a different node on lots of topics',
-      reason: 'HTTP clients cannot hold this many connections open'
-    }].concat(
-      isWindows
-        ? [{
-            name: 'should send/receive 100 messages',
-            reason: 'FIXME https://github.com/ipfs/interface-ipfs-core/pull/188#issuecomment-354673246 and https://github.com/ipfs/go-ipfs/issues/4778'
-          },
-          {
-            name: 'should receive multiple messages',
-            reason: 'FIXME https://github.com/ipfs/interface-ipfs-core/pull/188#issuecomment-354673246 and https://github.com/ipfs/go-ipfs/issues/4778'
-          }]
-        : []
-    )
+    skip: isWindows
+      ? [{
+          name: 'should send/receive 100 messages',
+          reason: 'FIXME https://github.com/ipfs/interface-ipfs-core/pull/188#issuecomment-354673246 and https://github.com/ipfs/kubo/issues/4778'
+        },
+        {
+          name: 'should receive multiple messages',
+          reason: 'FIXME https://github.com/ipfs/interface-ipfs-core/pull/188#issuecomment-354673246 and https://github.com/ipfs/kubo/issues/4778'
+        }]
+      : []
   })
 
   tests.repo(commonFactory)
 
   tests.stats(commonFactory)
 
   tests.swarm(commonFactory)
+}
+
+describe('kubo-rpc-client tests against kubo', function () {
+  const goVersionsToTest = ['current']
+  /**
+   * Enable running the tests across all go-ipfs versions listed below.
+   * Works best with `--bail=false` otherwise the tests will stop at the first
+   * failure.
+   */
+  if (process.env.GO_IPFS_GAMUT != null) {
+    goVersionsToTest.push(...['12.0', '12', '13', '14', '15', '16'])
+  }
```

Comment:
Should we remove this, alongside with the dependencies for the older versions?

Reply:
sure, but I can imagine it being useful when upgrading to go-ipfs 18